ext/session/tests/user_session_module/bug60634_error_4.phpt

```diff
@@ -14,34 +14,30 @@ ob_start();
 function open($save_path, $session_name) {
     return true;
 }
-
 function close() {
     echo "close: goodbye cruel world\n";
     exit;
 }
-
 function read($id) {
     return '';
 }
-
 function write($id, $session_data) {
     echo "write: goodbye cruel world\n";
     throw new Exception;
 }
-
 function destroy($id) {
     return true;
 }
-
 function gc($maxlifetime) {
     return true;
 }
-
 session_set_save_handler('open', 'close', 'read', 'write', 'destroy', 'gc');
+
 session_start();
 
 ?>
 --EXPECTF--
+Deprecated: Calling session_set_save_handler() with more than 2 arguments is deprecated in %s on line %d
 write: goodbye cruel world
```

Comment:
~This test can also be converted to use a class~ The error 5 variant is the OOP one...